### PR TITLE
send responders list and timeline entries to google script

### PR DESCRIPTION
**Pull request by lexelby**
I also added documentation for the google script stuff and included a copy of the script in the repo.

### CODE REVIEW

File: docs/google-script.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+Creating a Google Doc programmatically is a somewhat convoluted process.  Advanced document manipulation can only be done by a [Google Apps Script](https://developers.google.com/apps-script/) that runs on Google's servers.  This script [exposes](https://developers.google.com/apps-script/guides/rest/) an API that Retrodot can call to create a retrospective notes document.
+
+Here is the overall flow:
+
+1. User clicks "prepare retro" in the Retrodot admin interface.
+1. Retrodot calls a function in the Google Apps Script to prepare a new retrospective notes document.
+1. The script finds the template document by looking in Drive for a document named "Incident Retrospective Template".
+1. Retrodot performs a few search/replace operations:
+* `<incident_id>` is replaced with the incident number.
+* `<incident_title>` is repalced with the incident title.
```
Comment (joshuatobin) [inline docs/google-script.md]: typo for `replaced`

Comment (lexelby) [inline docs/google-script.md]: thanks! fixed in 4e1cbc4

**Comment by joshuatobin** [comment]
@lexelby Looks good except for the rubocop errors

**Comment by lexelby** [comment]
@joshuatobin Fixed those in 7d5f00c.  Did I miss any?

**Comment by joshuatobin** [comment]
👍

**Comment by lexelby** [comment]
@joshuatobin as a note, I had to create a new google script because it won't let me deploy a new version of yours.  I'll update the config var as I deploy this to staging and production.  Going to test the review app first.

**Comment by lexelby** [comment]
Bleh, can't test the review app because SSL won't work.  I'll have to push this to staging and test there.

**Pull request merged**